### PR TITLE
Bug 1996783: Bump to Go v1.16 and OCP v4.9

### PR DESCRIPTION
**Pull request by pierreprinetti**

### CODE REVIEW

**Comment by pierreprinetti** [comment]
This is now ready for review.
/cc mandre mdbooth

File: images/manila-provisioner/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM openshift/origin-release:golang-1.10
```
Comment (mandre) [inline images/manila-provisioner/Dockerfile:1]: This image is apparently never used and can potentially be removed:
```
❯ ag --stats "images/manila-provisioner/Dockerfile\s"
0 matches
```

**Comment by pierreprinetti** [comment]
Good catch; I am removing the image for the deprecated Manila provisioner.

**Comment by EmilienM** [comment]
/lgtm
/hold

Feel free to unhold when CI is happier.
Note that the Cinder job will fail on 4 tests, related to Snapshot. Manila will fail on one test, but the rest should work. We're working on this into separated PRs.

**Comment by EmilienM** [comment]
The errors on `e2e-openstack-csi-cinder` are known, feel free to ignore them.

**Comment by pierreprinetti** [comment]
/hold cancel

Thanks Emilien, canceling the hold following your precious advice.

**Pull request merged**